proc_CPMG.py

```diff
@@ -3,29 +3,76 @@
 from proc_scripts import *
 from proc_scripts import postproc_dict
 from proc_scripts.fitting import decay
+from sympy import symbols
+import sympy as sympy
 fl = fl_mod()
 logger = init_logging('info')
-for searchstr, exp_type, nodename, postproc, label_str, f_range in [
-        #('200221_CPMG_TEMPOLgel_2p9_1','test_equip','signal','spincore_CPMG_v1','deadtime=5',(-500,500)),
-        #('200304_CPMG_2p6_1','test_equip','signal','spincore_CPMG_v1','deadtime=5',(-500,500)),
-        #('200305_CPMG_3p5_2','test_equip','signal','spincore_CPMG_v1','deadtime=5',(-500,500)),
-        #('200305_CPMG_3p6_2','test_equip','signal','spincore_CPMG_v1','deadtime=5',(-500,500)),
-        ('200305_CPMG_3p7_2','test_equip','signal','spincore_CPMG_v1','deadtime=5',(-500,500)),
-        #('200305_CPMG_3p7_3','test_equip','signal','spincore_CPMG_v1','deadtime=5',(-500,500)),
-        #('200305_CPMG_3p8_2','test_equip','signal','spincore_CPMG_v1','deadtime=5',(-500,500)),
-        #('200305_CPMG_3p9_2','test_equip','signal','spincore_CPMG_v1','deadtime=5',(-500,500)),
-        #('200305_CPMG_4p0_1','test_equip','signal','spincore_CPMG_v1','deadtime=5',(-500,500)),
+for searchstr, exp_type, nodename, postproc, label_str, f_range, spincore in [
+        ('w8_200917','test_equip',6,'ag_CPMG_strob','water loading 8',(-500,500),False),
+        #('200221_CPMG_TEMPOLgel_2p9_1','test_equip','signal','spincore_CPMG_v1',
+        #    'deadtime=5',(-500,500),True),
+        #('freeSL_201007','test_equip',3,'ag_CPMG_strob','free SL',(-200,200),False),
+        #('200305_CPMG_3p5_2','test_equip','signal','spincore_CPMG_v1',
+        #    'deadtime=5',(-500,500),True),
+        #('200305_CPMG_3p6_2','test_equip','signal','spincore_CPMG_v1',
+        #    'deadtime=5',(-500,500),True),
+        #('200305_CPMG_3p7_2','test_equip','signal','spincore_CPMG_v1',
+        #    'deadtime=5',(-500,500),True),
+        #('200305_CPMG_3p7_3','test_equip','signal','spincore_CPMG_v1',
+        #    'deadtime=5',(-500,500),True),
+        #('200305_CPMG_3p8_2','test_equip','signal','spincore_CPMG_v1',
+        #    'deadtime=5',(-500,500),True),
+        #('200305_CPMG_3p9_2','test_equip','signal','spincore_CPMG_v1',
+        #    'deadtime=5',(-500,500),True),
+        #('200305_CPMG_4p0_1','test_equip','signal','spincore_CPMG_v1',
+        #    'deadtime=5',(-500,500),True),
         ]:
-    s =  find_file(searchstr, exp_type=exp_type,
-            expno=nodename, postproc=postproc, lookup=postproc_dict)
+    s = find_file(searchstr, exp_type=exp_type,
+            expno=nodename, postproc=postproc, lookup=postproc_dict, fl=fl)
+    s.ift('t2')
+    if spincore:
+        s.reorder('nScans',first=True)
+        s = s['ph1',1]
+        s.mean('nScans')
+        s.reorder('t2',first=True)
```

Comment:
why are you selecting different coherence pathways in the two cases?  (add as comment)

Reply:
this isn't spincore vs. not, it's whether or not the second pulse has been phase cycled.  You should be able to do something like `if 'ph2' in s.dimlabels` which would be the non-spincore option and `else` which would be the spincore option.  Similarly, `if 'nScans' in s.dimlabels: s.mean('nScans')`, rather than making it part of this "spincore" conditional